src/var/binary_snps.jl

```diff
@@ -0,0 +1,167 @@
+
+#=
+Objects of type BinarySNPs store the SNPs of an individual
+The name BinarySNPs is double edged: it refers to both the fact that
+SNPs are stored in a binary format, and also that the SNPs are binary,
+in that they have two states, a reference state and an alternate state.
```

Comment:
How are hets handled here? I'm assuming that the rows of this matrix are *alleles* rather than SNPs, and that the Index work is used to select the N alleles for each SNP?